### PR TITLE
Feature/caching

### PR DESCRIPTION
**Pull request by Klemmensen**
## Description

- Updates cache settings in NGINX

## References

https://github.com/Energinet-DataHub/energy-origin-issues/issues/405

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -51,6 +51,7 @@
   },
   "private": true,
   "dependencies": {
+    "@angular-devkit/core": "12.2.16",
```
Comment (LayZeeDK) [inline package.json:54]: issue: revert this change

**Review by LayZeeDK — DISMISSED** [review]

**Comment by LayZeeDK** [comment]
With private browser tab open:
- [ ] Open feature environment
- [ ] Refresh, verify 304 responses
- [ ] Deploy changed JSON and image files
- [ ] Refresh, verify fresh 200 responses
- [ ] Deploy release without JSON and image file changes
- [ ] Refresh, verify 304 responses

**Review by Chrisbh — CHANGES_REQUESTED** [review]

File: infrastructure/eo/host/nginx.conf [hunk 2]
```diff
@@ -34,9 +34,21 @@ server {
...
-  # files
-	location ~ \.(?!html) {
-		add_header Cache-Control "public, max-age=2678400"; # cache for one month
+  # Static files
+  # We target everything except .html files
+	location ~ \.(?!html$) {
 		try_files $uri =404; # 404 if file doesn't exist
+
+    # In order not to spam the access log, it is turned off except for 404 not found errors
+		access_log    off;
```
[the remark this answers is not in the document]
Comment (Chrisbh) [reply infrastructure/eo/host/nginx.conf:43]: Makes sense, thanks for the explanation. 

I would not usually accept having mixed tabs and spaces, I would go with one of them and stick to that. I am not sure what the standard is in this repository though.

[the remark this answers is not in the document]
Comment (Klemmensen) [reply infrastructure/eo/host/nginx.conf:43]: Sure NP - Could you help me check again for the tabs, I converted the file to use tabs - Hopefully you see that too then?

**Review by Chrisbh — APPROVED** [review]

**Pull request merged**